src/pinball_wizard.coffee

```diff
@@ -135,8 +135,9 @@ define ->
     JSON.parse(storage.getItem('pinball_wizard') or "[]") or []
 
   activatePermanently = (name) ->
-    appendPermanent(name)
-    activate(name, 'permanent')
+    for n in name.split(',')
```

Comment:
Yeah, it does take more characters (2), but it's odd to pass in a comma delimited string when working with the lib directly.